hw-02/src/helpers/get-route-handler.js

```diff
@@ -0,0 +1,19 @@
+const getIdFreeUrl = url => {
+    const lastIndex = url.lastIndexOf('/');
+    const idString = url.slice(lastIndex +1);
+    
+    if (url.slice(0, lastIndex) === "/products" && idString && lastIndex !== -1) {

```

Comment:
its very long condition

create variable and call it to the same meaning what this contitions do 

for example

const isUserEmailValid = user && user.email && user.name.includes('@') !== -1